tailscale/install.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+# Copyright (c) 2022 Tailscale Inc & AUTHORS All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+set -euo pipefail
+
+tailscale_url="https://pkgs.tailscale.com/stable/tailscale_${VERSION}_amd64.tgz"
+
+download() {
+  if command -v curl >& /dev/null; then
+    curl -fsSL "$1"
+  elif command -v wget >& /dev/null; then
+    wget -qO - "$1"
+  else
+    echo "Must install curl or wget to download $1" 1>&2
+    return 1
+  fi
+}
+
+script_dir="$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+scratch_dir="/tmp/tailscale"
+mkdir -p "$scratch_dir"
+trap 'rm -rf "$scratch_dir"' EXIT
```

Comment:
I don't understand this line: it seems like it deletes the `/tmp/tailscale` directory which the previous line should have just created.